README.md

```diff
@@ -11,6 +9,27 @@ over the years.
 
 [stylelint-config-standard]: https://github.com/stylelint/stylelint-config-standard
 
+## Project goals
+
+This project was developed with a number of goals in mind, these should be
+considered whenever proposals are made to add to or amend the rules.
+
+1. Lean on community standards over defining our own rules as much as possible,
+   it's not a good use of our time to debate preferences so lets follow [common
+   community guidelines][stylelint-config-standard] unless we have a compelling
+   reason not to do so.
+1. Be consistent with [GOV.UK design-system][] patterns and conventions, most
+   of the initial rules of this project were defined to maintain consistency.
+1. Reflect modern CSS practices over providing compatibility for legacy
+   projects, if it is unfeasible for legacy projects to adopt all these rules
+   they are encouraged to disable problematic rules in their project
+   configuration.
+1. Support CSS and SASS equally.
```

Comment:
We mention SASS here, and SCSS later on - to avoid confusion we should be consistent in which name we use.